src/framework/xr/xr-manager.js

```diff
@@ -746,6 +747,29 @@ class XrManager extends EventHandler {
             view.viewInvMat.set(viewRaw.transform.matrix);
         }
 
+        // update the camera fov properties only when we had 0 views
+        if (lengthOld === 0 && this.views.length > 0) {
+            const viewProjMat = new Mat4();
+            const view = this.views[0];
+
+            viewProjMat.copy(view.projMat);
+            const data = viewProjMat.data;
+
+            const fov = (2.0 * Math.atan(1.0 / data[5]) * 180.0) / Math.PI;
+            const aspectRatio = data[5] / data[0];
+            const farClip = data[14] / (data[10] + 1);
+            const nearClip = data[14] / (data[10] - 1);
+
+            const camera = this._camera.camera;
+            camera._xrAspectRatio = aspectRatio;
```

Comment:
ideally we don't access private properties, but create some function on the Camera class for this, maybe `setXrProperties` or similar.

Reply:
internally it would also do `_projMatDirty = true;`